multihost_test/Bugzillas/test_regression.py

```diff
@@ -62,3 +62,38 @@ def test_bz_639975(self, multihost):
         multihost.client[0].run_command("userdel -rfZ user_11")
         cmd = multihost.client[0].run_command("semanage login -l")
         assert 'user_11' not in cmd.stdout_text
+
+    @pytest.mark.tier1
+    def test_bz_1220504(self, multihost):
+        """
+        :title: BZ#1220504 (usermod -p allowing colon (ie. '' ) in encrypted)
+        :id: 7d7848f8-9de1-11ed-b5f3-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1220504
+        :steps:
+          1. Create test user
+          2. Modify user password with usermod and include colon
+          3. Now see that the ':' was accepted even though it is
+            really a delimiter for the file
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should not succeed
+        """
+        user = "bz1220504"
+        client = multihost.client[0]
+        client.run_command("cp -vf /etc/shadow /etc/shadow_anuj")
+        client.run_command(f"useradd {user}")
+        # Check if the usermod -p does report error when pasword contains : (colon)
+        with pytest.raises(subprocess.CalledProcessError):
+            client.run_command(f"usermod -U {user}")
```

Comment:
The comment (usermod -p) and the actual command executed (usermod -U) don't match.

Reply:
I was porting this script :

https://pkgs.devel.redhat.com/cgit/tests/shadow-utils/tree/Regression/bz1220504-usermod-p-allowing-colon-ie-in-encrypted/runtest.sh

This is original script .

Reply:
That seems like a general comment for the complete test, so I'd suggest you to move it before the test in the python code.

Reply:
That seems like a general comment for the complete test, so I'd suggest you to move it before the test in the python code.